src/components/contribute/index.js

```diff
@@ -479,6 +488,17 @@ export class Contribute extends React.Component {
       'qr-selected': contributeThrough === CONTRIBUTION_OPTIONS.QR
     })
 
+    const crowdsaleAddress = (crowdsale && crowdsale.execID) || (MintedCappedProxy && MintedCappedProxy.addr)
+    const crowdsaleAddressTruncated =
+      (crowdsale && truncateStringInTheMiddle(crowdsale.execID)) ||
+      (MintedCappedProxy && truncateStringInTheMiddle(MintedCappedProxy.addr))
```

Comment:
@mariano-aguero let's truncate only for execution id. I mean not truncate in the middle, if it is a proxy address:
```
const crowdsaleAddressTruncated =
      (crowdsale && truncateStringInTheMiddle(crowdsale.execID)) ||
      (MintedCappedProxy && MintedCappedProxy.addr)
```

Reply:
Hi @vbaranov , changes made. Thanks for the feedback